### PR TITLE
Pop up windows

### PR DESCRIPTION
**Pull request by ulisesedu7**
- In this project, I've implemented the pop-up windows for a more detailed description of each project. 
- I've used even listeners in js to check which btn is clicked and then change the information accordingly. 
- Finally, I've changed the media queries to adjust the pop-up window for the desktop version.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -433,6 +478,65 @@ Desktop hidden stuff
   display: none;
 }
 
+/*
+Pop Up windows Styles
```
Comment (Ismailco) [inline styles.css:487]: - [x] The close button color is different from the Figma design, please consider changing it to ```#67798E``` so your design be is as close to the Figma design as possible

File: styles.css [hunk 2]
```diff
@@ -433,6 +478,65 @@ Desktop hidden stuff
...
+  background: #c1c7d0;
+  opacity: 0;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+  transition: opacity 0.3s ease;
+  overflow: scroll;
+}
+
+.mobile-pop {
```
Comment (Ismailco) [inline styles.css:502]: - [x] The popup window is too big please consider giving it a smaller ```width``` and ```height``` so it is closer to the given design as possible

- Your design:
Screenshot_20220512_185634

- Figma design:
Screenshot_20220512_185600

File: javascript/data.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  id: 6,
+  featureImg: 'project-img6.svg',
+  projectName: 'Project 6',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsu',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const projectInfo = [project1, project2, project3, project4, project5, project6];
```
Comment (Ismailco) [inline javascript/data.js]: - [x] For best practices please consider adding the objects directly inside the array

File: index.html [hunk 4]
```diff
@@ -40,42 +45,26 @@
...
+          <li><a><img src="images/twitter.png" alt="twitter icon" /></a></li>
+          <li><a><img src="images/linkedin-icon.png" alt="linkedin icon" /></a></li>
+          <li><a><img src="images/medium.png" alt="A big M icon" /></a></li>
+          <li><a><img src="images/github.png" alt="github icon" /></a></li>
+          <li><a><img src="images/angellist.png" alt="two tumbs up icon" /></a></li>
         </ul>
       </section>
     </div>
 
     <section class="works-section" id="portafolio">
```
Comment (Ismailco) [inline index.html:62]: - [x] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ulisesedu7,
Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (ulisesedu7) [reply index.html:62]: @Ismailco Hello!! Thank you so much for the review, would you mind clarifying more about this part? I'm not sure if you mean to remove the images, or what exact part has to be created dynamically, I'd appreciate it a lot!

Comment (Ismailco) [reply index.html:62]: @ulisesedu7, as a requirement you need to remove the whole project section from your HTML file and instead Create it dynamically using JavaScript

Comment (ulisesedu7) [reply index.html:62]: @Ismailco Thanks for clarifying, just to know if I'm getting it right, you mean the project section pop-up window, right? Does that have to be removed from the HTML to be created with JS instead?

Comment (Ismailco) [reply index.html:62]: @ulisesedu7, not the popup window, I mean the projects section from line ```62``` to line ```175``` in your HTML file needs to be removed and created using JavaScript

Comment (ulisesedu7) [reply index.html:62]: Gotcha Thanks @Ismailco!

**Review by smunozmo — APPROVED** [review]
Hi @ulisesedu7, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**